trucks_project/tests/unit/test_client_manager.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from truck_delivery_fernando.db_redis_connector import DBRedisConnector
+from truck_delivery_fernando.client_manager import ClientManager
+
+
+@pytest.fixture
+def client_manager(mocker):
+    object_to_save = {"Client1": {"name": "test1"}}
+    object_to_return = {"Client1": {"name": "test1"}}
+    list = [{"Client1": {"name": "test2"}}, {"Client2": {"name": "test2"}}, {"Client3": {"name": "test3"}}]
+    mocker.patch("truck_delivery_fernando.client_manager.ClientManager.save_document").return_value = object_to_save
```

Comment:
The db_connector methods should be mocked, as you are trying to test the ClientManager methods

Reply:
Hi @favipcj, I already fix the test client manager. Please check it